apps/tests/devapps/managedidentity/managedidentity_sample.go

```diff
@@ -3,31 +3,46 @@ package main
 import (
 	"context"
 	"fmt"
+	"log"
 
 	mi "github.com/AzureAD/microsoft-authentication-library-for-go/apps/managedidentity"
 )
 
 func runIMDSSystemAssigned() {
+
 	miSystemAssigned, err := mi.New(mi.SystemAssigned())
 	if err != nil {
-		fmt.Println(err)
+		log.Fatal(err)
 	}
-	result, err := miSystemAssigned.AcquireToken(context.Background(), "https://management.azure.com/")
+	result, err := miSystemAssigned.AcquireToken(context.TODO(), "https://management.azure.com")
 	if err != nil {
-		fmt.Println(err)
+		log.Fatal(err)
 	}
 	fmt.Println("token expire at : ", result.ExpiresOn)
+	fmt.Println("token source : ", result.Metadata.TokenSource)
```

Comment:
Should an application developer care about the `TokenSource`? Is `token source : 1` useful output?